### PR TITLE
ssh banner: Add tests for ssh banner

### PR DESCRIPTION
**Pull request by TheKharleeci**
Bug: 4903

Previous PR: #611

### CODE REVIEW

File: tests/bug-4903-02/test.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# *** Add configuration here ***
```
Comment (inashivb) [inline tests/bug-4903-02/test.yaml:1]: nit: Please remove this

File: tests/bug-4903-03/test.yaml [hunk 2]
```diff
@@ -0,0 +1,112 @@
+# *** Add configuration here ***
```
Comment (inashivb) [inline tests/bug-4903-03/test.yaml:1]: nit

File: tests/bug-4903-04/test.yaml [hunk 3]
```diff
@@ -0,0 +1,116 @@
+# *** Add configuration here ***
```
Comment (inashivb) [inline tests/bug-4903-04/test.yaml:1]: nit

File: tests/bug-4903-01/test.yaml [hunk 4]
```diff
@@ -0,0 +1,56 @@
+# *** Add configuration here ***
```
Comment (inashivb) [inline tests/bug-4903-01/test.yaml:1]: nit

**Review by inashivb — CHANGES_REQUESTED** [review]

**Comment by TheKharleeci** [comment]
Followed by: #643

**Pull request closed without merging**